manager/controllers/systems_test.go

```diff
@@ -36,6 +36,7 @@ func TestSystemsDefault(t *testing.T) {
 	assert.Equal(t, "RHEL", output.Data[0].Attributes.OSName)
 	assert.Equal(t, "8", output.Data[0].Attributes.OSMajor)
 	assert.Equal(t, "1", output.Data[0].Attributes.OSMinor)
+	assert.Equal(t, "8.1", output.Data[0].Attributes.RHSMVersion)
```

Comment:
Please, update also advisory_systems_test.go and system_detail_test.go. Similarly like os version is done.